packages/docs-app/src/examples/core-examples/buttonExamples.tsx

```diff
@@ -0,0 +1,175 @@
+    const code = `<Button text="This is a very long button label that will be truncated" ellipsizeText={true} />`;
+    return (
+        <CodeExample code={code} {...props}>
+            <Button text="This is a very long button label that will be truncated" ellipsizeText={true} />
+        </CodeExample>
+    );
+};
+
+export const ButtonIconWithTextExample: React.FC<ExampleProps> = props => {
+    const code = `<Button icon="refresh" intent="danger" text="Reset" />
+<Button icon="user" rightIcon="caret-down" text="Profile settings" />
+<Button rightIcon="arrow-right" intent="success" text="Next step" />
+<Button>
+    <Icon icon="document" /> Upload... <Icon icon="small-cross" />
+</Button>`;
+    return (
+        <CodeExample code={code} {...props}>
+            <Button icon="refresh" intent="danger" text="Reset" />
+            <Button icon="user" rightIcon="caret-down" text="Profile settings" />
+            <Button rightIcon="arrow-right" intent="success" text="Next step" />
+            <Button>
+                <Icon icon="document" /> Upload... <Icon icon="small-cross" />
+            </Button>
+        </CodeExample>
+    );
+};
+
+export const ButtonIconExample: React.FC<ExampleProps> = props => {
+    const code = `<Button icon="edit" aria-label="edit" />
+<Button icon="share" outlined={true} aria-label="share" />
+<Button icon="filter" intent="primary" minimal={true} aria-label="filter" />
+<Button icon="add" intent="success" aria-label="add" />
+<Button icon="trash" disabled={true} intent="danger" aria-label="delete" />`;
+    return (
+        <CodeExample code={code} {...props}>
+            <Button icon="edit" aria-label="edit" />
+            <Button icon="share" outlined={true} aria-label="share" />
+            <Button icon="filter" intent="primary" minimal={true} aria-label="filter" />
+            <Button icon="add" intent="success" aria-label="add" />
+            <Button icon="trash" disabled={true} intent="danger" aria-label="delete" />
+        </CodeExample>
+    );
+};
+
+export const ButtonStatesExample: React.FC<ExampleProps> = props => {
+    const code = `<Button text="Default" />
+<Button text="Active" active={true} />
+<Button text="Disabled" disabled={true} />
+<Button text="Loading..." loading={true} />`;
+    return (
+        <CodeExample code={code} {...props}>
+            <Button text="Default" />
+            <Button text="Active" active={true} />
+            <Button text="Disabled" disabled={true} />
+            <Button text="Loading..." loading={true} />
+        </CodeExample>
+    );
+};
+
+export const ButtonAnchorButtonExample: React.FC<ExampleProps> = props => {
+    const code = `<AnchorButton href="#core/components/buttons.anchorbutton" rightIcon="share" text="Link" />`;
+    return (
+        <CodeExample code={code} {...props}>
+            <AnchorButton href="#core/components/buttons.anchorbutton" rightIcon="share" text="Link" />
```

Comment:
I think we should include a full url here, something like 
```suggestion
            <AnchorButton href="https://blueprintjs.com/" rightIcon="share" text="Link" />
```

As this will help make it clear for new users what this means.